src/crimson/osd/ops_executer.h

```diff
@@ -616,7 +616,10 @@ OpsExecuter::RollbackHelper<Func>::rollback_obc_if_modified(
     ox->obc->get_oid(),
     e,
     need_rollback);
-  return need_rollback ? func(*ox->obc) : interruptor::now();
+  if (need_rollback) {
+    func(ox->obc);
+  }
+  return interruptor::now();
```

Comment:
Can `rollback_obc_if_modified` also be voided?
We could simply call it instead of chaining when needed, right?

Reply:
Ok, will change it, too:-)

Reply:
Thanks, I'll await with the next batch to include this one too.